src/coprocessor/endpoint.rs

```diff
@@ -660,14 +666,6 @@ mod tests {
     use std::time::Duration;
 
     #[test]
-    fn test_get_req_type_str() {
```

Comment:
Why remove the case?

Reply:
Since function `get_req_type_str ` has been removed @BusyJay

Reply:
But there is new function named `get_scan_tag`.